Makefile

```diff
@@ -34,7 +34,7 @@ check-release:
 	@echo Released srclib $(V)
 
 install-std-toolchains:
```

Comment:
@slimsag right, I meant "std" is kinda a deprecated term; this is more like `install-all-toolchains`